Lombiq.HelpfulExtensions/Extensions/ContentSets/Workflows/Handlers/WorkflowContentSetEventHandler.cs

```diff
@@ -54,14 +54,16 @@ public class WorkflowContentSetEventHandler : IContentSetEventHandler
                 case ContentSetLinkViewModel viewModel:
                     links.Add(viewModel);
                     break;
-                case IEnumerable<object> collection when collection.CastWhere<ExpandoObject>() is { } objects && objects.Any():
-                    links.AddRange(JToken.FromObject(objects).ToObject<IEnumerable<ContentSetLinkViewModel>>());
-                    break;
                 case ExpandoObject expandoObject:
-                    links.Add(JToken.FromObject(expandoObject).ToObject<ContentSetLinkViewModel>());
+                    links.Add(SerializeAndDeserialize<ContentSetLinkViewModel>(expandoObject));
+                    break;
+                case IEnumerable<object> collection when
+                    collection.CastWhere<ExpandoObject>().ToList() is { } objects &&
+                    objects.Count != 0:
+                    links.AddRange(SerializeAndDeserialize<IEnumerable<ContentSetLinkViewModel>>(objects));
```

Comment:
Why do we need this?

Reply:
It was already there [a few lines above](https://github.com/Lombiq/Helpful-Extensions/pull/161/files/5800d7b01784a9d107f18207a563e3e4d18caf3c#diff-cde8ef6bc1c3fa66dfdfb10a159625b6808706184a53db3cbd3ad9982bd37a84L58).

Reply:
I see. You are right.